ryan_savill/02_intro_to_skimage/readme.md

```diff
@@ -0,0 +1,288 @@
+# Introduction to Image Analysis Basics in Python with Scikit Image
+Now that we have a grasp of the basics of python it's time to get started with some proper image analysis! For the purposes of trying out image analysis I have a picture of a tribolium embryo with stained nuclei. It previously was a 3D image but we are working with a maximum projection to keep it simple (This means the maximum intensity over all slices of the embryo were taken). Now to see how processing is done in python the explanation will be shown with this example.
+The first step in any program is to import the libraries needed. You can import whole libraries for example numpy:
+
+
+```python
+# we are importing it as 'np' to make the 
+# code that we have to write shorter
+import numpy as np
+```
+
+Or we can import parts of a library that we need for example from scikit image or matplotlib
+
+
+```python
+from skimage import data, io, filters, feature
+import matplotlib.pyplot as plt
+```
+
+If you have downloaded anaconda and created an environment as was described in my first [post](https://biapol.github.io/blog/rs_Introduction_to_Python) you will have these libraries pre-installed as they are some of the standard scientific packages needed in python. If you get any errors, go to the command line, activate your environment and install the libraries. If you search the library you want to install in google you will find how to install them in your environment on the sites. 
+
+Now that we have the libraries imported we can talk a bit about them: Numpy is one of the most important libraries you'll need since a lot of the other libraries we will work with are partially based on numpy. In short numpy is a framework with which arrays can be used for mathmatical operations like vectors and matrices, but it is also far more than that. I would HIGHLY suggest watching [this video](https://www.youtube.com/watch?v=4uFs1qouPEI&list=PLZsOBAyNTZwYHBIlu_PUO19M7aHMgwBJr&index=12&ab_channel=DigitalSreeni) on numpy arrays and how they work. It is also helpful to play around with numpy arrays yourself to get used to how they work, because pictures in python are nothing else that numpy arrays of intensities!
```

Comment:
```suggestion
Now that we have the libraries imported we can talk a bit about them: Numpy is one of the most important libraries you'll need since a lot of the other libraries we will work with are partially based on numpy. In short, numpy is a framework with which arrays can be used for mathematical operations on vectors and matrices, but it is also far more than that. I would HIGHLY suggest watching [this video](https://www.youtube.com/watch?v=4uFs1qouPEI&list=PLZsOBAyNTZwYHBIlu_PUO19M7aHMgwBJr&index=12&ab_channel=DigitalSreeni) on numpy arrays and how they work. It is also helpful to play around with numpy arrays yourself to get used to how they work, because images in python are nothing else that numpy arrays of intensities!
```
minor commas and grammar